Add an production ready Dockerfile (#186)
Avaiga/taipy-doc: docs/manuals/deployment/linux/redhat.md

!!! Note

>     If you are using a `SQLDataNode^` based on Microsoft SQL Server, you need to install your corresponding

Reviewer: ```suggestion
    If you are using an SQL database based on Microsoft SQL Server, you need to install your corresponding
```
